QuickFIXn/Transport/StreamFactory.cs

```diff
@@ -50,7 +50,9 @@ public static class StreamFactory
             Socket socketThruProxy = new Socket(AddressFamily.InterNetwork, SocketType.Stream, ProtocolType.Tcp);
             socketThruProxy.Connect(proxyEndPoint);
 
-            string proxyMsg = $"CONNECT {destIp}:{destPort} HTTP/1.1 \n\n";
+            string proxyMsg = !string.IsNullOrWhiteSpace(destHostName)
```

Comment:
I see you made the change, thanks.

But you didn't answer the question--  Will the new string will work with non-Squid users?  (I assume it will, just covering my bases)